components/Projects/Project.tsx

```diff
@@ -12,35 +12,38 @@ export const Project = ({ title, description, tags, author }: ProjectProps) => {
   const handleContribute = () => {
     return;
   };
+  const [innerWidth, setInnerWidth] = useState(0);
+  useEffect(() => {
```

Comment:
Just curious can you explain to me what this piece of code is doing?

Reply:
it's getting the width of the window. `window` is not available in Server code so i've put it in `useEffect` to make sure we get the `window` property

Reply:
ok got it. it's a nice trick I come to know today, thanks!